docs/quick-start/new-project/README.md

```diff
@@ -0,0 +1,268 @@
+
+# Start a New Project
+## Project setup
+To start, create a new folder to store your project and then run `npm init` to create a new `package.json` file.
+
+Next, install Mira, the AWS CDK and other dependencies. Edit your `package.json` file to add the following:
+
+```json
+ "dependencies": {
+    "@aws-cdk/aws-cloudformation": "1.61.1",
+    "@aws-cdk/aws-lambda-event-sources": "1.61.1",
+    "@aws-cdk/aws-rds": "1.61.1",
+    "@aws-cdk/aws-s3-deployment": "1.61.1",
+    "@aws-cdk/aws-secretsmanager": "1.61.1",
+    "@aws-cdk/core": "1.61.1",
+    "aws-cdk": "1.61.1",
+    "mira": "^1.3.0"
+}
+```
+Note: Use the same version of the CDK for every additional library you choose. At the time of writing, Mira 1.3.0 supports the 1.61.1 version of the CDK. 
+
+Run `npm install`. 
+
+Mira and the AWS CDK are now installed.
+
+## Configure Mira
+Once you have Mira installed, you can invoke it via the `npx` command. For example, to view the built-in documentation run `npx mira docs`. This presents you with a local URL that you use in your browser.
+
```

Comment:
May be worth noting what NPX does here or why it was used?  e.g.:

Note about NPX: NPX is intended to be used with Mira being installed locally to an application's `node_modules` directory.  Mira relies on NPX to resolve cross-platform issues.  NPX will lookup `node_modules/bin/mira` and run that command from your application directory.

Reply:
Added